server/src/actors/events.rs

```diff
@@ -221,6 +247,7 @@ impl ProcessEvent {
     }
 }
 
+#[cfg_attr(not(feature = "processing"), allow(dead_code))]
```

Comment:
if it's dead code when processing is not enabled, why not wrap it in `cfg(feature=processing)`?

Reply:
One of the variants is `dead_code`, but they are not `unused`. Meaning, it's never constructed, but it is checked. 
Removing the check would just introduce more `cfg`s, so this was the solution with least code.